engine/dlib/src/dlib/hash.cpp

```diff
@@ -3,10 +3,10 @@
 // Copyright 2009-2014 Ragnar Svensson, Christian Murray
 // Licensed under the Defold License version 1.0 (the "License"); you may not use
 // this file except in compliance with the License.
-// 
+//
```

Comment:
I guess we should fix our update-script so we don't have to deal with this another full year.